### PR TITLE
Fixes - Loaders + Error management + Small fixes

### PR DESCRIPTION
**Pull request by cesar-tonnoir**

### CODE REVIEW

**Comment by cesar-tonnoir** [comment]
> need to build + update changelog

File: src/components/widget/widget.directive.coffee [hunk 1]
```diff
@@ -37,8 +37,6 @@ module.controller('ImpacWidgetCtrl', ($scope, $log, $q, $timeout, ImpacWidgetsSv
 
       (errorResponse) ->
         w.isLoading = false
```
Comment (alexnoox) [inline src/components/widget/widget.directive.coffee]: Maybe the `w.isLoading = false` should be in a finally bloc.

File: src/components/widgets-common/data-not-found/data-not-found.directive.coffee [hunk 2]
```diff
@@ -18,13 +17,14 @@ module.directive('commonDataNotFound', ($templateCache, $log, $http, ImpacAssets
...
-        $http.get(assetPath).then(
-          (success) ->
-            scope.bgImage = assetPath
-          (error) ->
-            $log.warn("Missing data-not-found image for #{scope.widgetEngine}")
-        )
+        image = _.find element.children().first().children(), (elem) ->
+          elem.id == 'not-found-bg'
+
+        image.onerror = ->
```
Comment (alexnoox) [inline src/components/widgets-common/data-not-found/data-not-found.directive.coffee]: Naming should be `onError`

File: src/components/widgets/sales-forecast/sales-forecast.directive.coffee [hunk 3]
```diff
@@ -53,7 +53,7 @@ module.controller('WidgetSalesForecastCtrl', ($scope, $q, ChartFormatterSvc, $fi
       period = null
       period = w.metadata.hist_parameters.period if w.metadata? && w.metadata.hist_parameters?
       formattedDates = _.map w.content.dates, (aDate) ->
-        $filter('mnoDate')(date, period)
+        $filter('mnoDate')(aDate, period)
```
Comment (alexnoox) [inline src/components/widgets/sales-forecast/sales-forecast.directive.coffee]: That must work way better :)

File: src/components/widget/widget.directive.coffee (hunk 1 above)
Comment (cesar-tonnoir) [inline src/components/widget/widget.directive.coffee]: :+1:

File: src/components/widgets-common/data-not-found/data-not-found.directive.coffee (hunk 2 above)
Comment (cesar-tonnoir) [inline src/components/widgets-common/data-not-found/data-not-found.directive.coffee]: nope: it's a built-in javascript callback ;)

**Pull request merged**